### PR TITLE
Simple Hello Microverse Project

### PR DESCRIPTION
**Pull request by Hope1226**
In this project I:

- Created GitHub repo
- Created one feature branch
- Created HTML/CSS/test files 
- Created .gitignore file 
- Created README

### CODE REVIEW

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @Hope1226 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] Please ensure that all your linter checks pass. It is important as it enables you to follow and adopt good coding practices. In your case, the stylint is failing.
- [x] Please ensure that your README is descriptive and that you are using the [template ](https://github.com/microverseinc/readme-template)provided by Microverse.
- [x] Please ensure that there is a .gitignore file listing test.md file and node_modules directory.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+
+## Getting Started
+
+N/A
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (safafa) [inline README.md:28]: - [ ] [OPTIONAL] I think it will look better if you replace Author1 with your name.

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+
+- GitHub: [@Hope1226](https://github.com/Hope1226)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [Умид Устабаев](https://www.linkedin.com/in/%D1%83%D0%BC%D0%B8%D0%B4-%D1%83%D1%81%D1%82%D0%B0%D0%B1%D0%B0%D0%B5%D0%B2-03b92b11a/)
```
Comment (safafa) [inline README.md:38]: - [ ] [OPTIONAL] I think it will look better if you remove Author2 section and only add it when it's needed.

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello @Hope1226  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**